src/m365/spo/commands/group/group-list.ts

```diff
@@ -2,11 +2,12 @@ import { AxiosRequestConfig } from 'axios';
 import { Logger } from '../../../../cli/Logger';
 import GlobalOptions from '../../../../GlobalOptions';
 import request from '../../../../request';
+import { odata } from '../../../../utils/odata';
 import { validation } from '../../../../utils/validation';
 import SpoCommand from '../../../base/SpoCommand';
 import commands from '../../commands';
 import { AssociatedGroupPropertiesCollection } from './AssociatedGroupPropertiesCollection';
-import { GroupPropertiesCollection } from './GroupPropertiesCollection';
```

Comment:
Do we still have need for this interface? I think this was the last location it was used. If that's the case, we can remove `GroupPropertiesCollection.ts`

Reply:
Correct!